packages/react-native-bridge/android/react-native-bridge/src/main/java/org/wordpress/mobile/WPAndroidGlue/GutenbergProps.kt

```diff
@@ -49,6 +50,7 @@ data class GutenbergProps @JvmOverloads constructor(
         putBoolean(PROP_CAPABILITIES_XPOSTS, enableXPosts)
         putBoolean(PROP_CAPABILITIES_CONTACT_INFO_BLOCK, enableContactInfoBlock)
         putBoolean(PROP_CAPABILITIES_LAYOUT_GRID_BLOCK, enableLayoutGridBlock)
+        putBoolean(PROP_CAPABILITIES_TILED_GALLERY_BLOCK, enableTiledGalleryBlock)
```

Comment:
I believe the following line needs to be added to https://github.com/WordPress/gutenberg/blob/trunk/packages/react-native-editor/android/app/src/main/java/com/gutenberg/MainActivity.java in order to tie things up and get this to work with the Android demo app:

`capabilities.putBoolean(GutenbergProps.PROP_CAPABILITIES_TILED_GALLERY_BLOCK, true);`

Reply:
🙈 I completely overlooked testing these PRs on Android, sorry about that. Thanks for pointing out what was missing here, I've now added it in bbdf59caf6417d538378d7c76eb5c6cf4e4060f6.